### PR TITLE
Add an option to use rules for payee matching

### PR DESCRIPTION
**Pull request by ashish01**

### CODE REVIEW

File: ledgerautosync/cli.py [hunk 1]
```diff
@@ -452,6 +458,12 @@ def run(args=None, config=None):
         "XDG_CONFIG_HOME", os.path.join(os.path.expanduser("~"), ".config")
     )
 
+    if args.rules and os.path.exists(args.rules):
+        with open(args.rules) as f:
+            for line in f:
+                regex, account = line.strip().split("\t")
```
Comment (egh) [inline ledgerautosync/cli.py:464]: I guess you can't split by whitespace?

Comment (ashish01) [reply ledgerautosync/cli.py:464]: Yeah, space can be part of both account name and expense description so used tab as a out of band character.

Another option is to use pythons CSV reader class. Let me know if that is more preferable.

Comment (egh) [reply ledgerautosync/cli.py:464]: Up to you - if you're happy with this it works for me :)

Comment (ashish01) [reply ledgerautosync/cli.py:464]: LGTM :)

**Pull request merged**